src/Cutscenes.ts

```diff
@@ -1914,6 +1914,144 @@ export class Cutscenes<TEightBittr extends FullScreenPokemon> extends EightBittr
     }
 
     /**
+     * Cutscene for a Gust attack in battle.
+     * 
+     * @param settings   Settings used for the cutscene.
+     * @param args   Settings for the routine.
+     */
+    public cutsceneBattleAttackGust(settings: IBattleCutsceneSettings, args: IBattleAttackRoutineSettings): void {
+        const attackerName: string = args.attackerName;
+        const defenderName: string = args.defenderName;
+        const defender: IThing = this.EightBitter.BattleMover.getThing(defenderName) as IThing;
+        const direction: number = attackerName === "player" ? 1 : -1;
+        const menu: IMenu = this.EightBitter.MenuGrapher.getMenu("BattleDisplayInitial") as IMenu;
+
+        const gusts: IThing[] = [
+            this.EightBitter.ObjectMaker.make("ExplosionSmall"),
+            this.EightBitter.ObjectMaker.make("ExplosionLarge"),
+            this.EightBitter.ObjectMaker.make("ExplosionSmall"),
+            this.EightBitter.ObjectMaker.make("ExplosionLarge"),
+            this.EightBitter.ObjectMaker.make("ExplosionSmall"),
+            this.EightBitter.ObjectMaker.make("ExplosionLarge"),
+            this.EightBitter.ObjectMaker.make("ExplosionSmall"),
+            this.EightBitter.ObjectMaker.make("ExplosionLarge")
+        ];
+        let gustX: number[] = [];
+        let gustY: number[] = [];
+        if (direction === -1) {
+            // Enemy use
+        } else {
+            gustX[0] = menu.left + (defender.width) * this.EightBitter.unitsize;
+            gustY[0] = menu.bottom - (defender.height) * this.EightBitter.unitsize;
+            gustX[1] = gustX[0] + 4 * this.EightBitter.unitsize;
+            gustY[1] = gustY[0];
+            gustX[2] = gustX[1] + 4 * this.EightBitter.unitsize;
+            gustY[2] = gustY[1];
+            gustX[3] = gustX[2] + 4 * this.EightBitter.unitsize;
+            gustY[3] = gustY[2];
+            gustX[4] = gustX[3] + 4 * this.EightBitter.unitsize;
+            gustY[4] = gustY[3];
+            gustX[5] = gustX[4] + 4 * this.EightBitter.unitsize;
+            gustY[5] = gustY[4] - 4 * this.EightBitter.unitsize;
+            gustX[6] = gustX[5] + 4 * this.EightBitter.unitsize;
+            gustY[6] = gustY[5] - 4 * this.EightBitter.unitsize;
+            gustX[7] = gustX[6] + 4 * this.EightBitter.unitsize;
+            gustY[7] = gustY[6] - 4 * this.EightBitter.unitsize;
+        }
```

Comment:
I think you could make this more elegant by storing differences in x and y positions instead of the hardcoded differences.